dockers/docker-fpm-quagga/bgpcfgd

```diff
@@ -157,7 +157,8 @@ class BGPConfigManager(object):
                     self.lo_ipv6 = ip_addr
                     txt = self.zebra_set_src_template.render(rm_name="RM_SET_SRC6", lo_ip=ip_addr, ip_proto="v6")
                 else:
-                    syslog.syslog(syslog.LOG_ERR, "Got ambigous ip addres '%s'" % ip_addr)
+                    syslog.syslog(syslog.LOG_ERR, "Got ambiguous ip address '%s'" % ip_addr)
```

Comment:
only typo changes?

Reply:
There were return added after typo change in the PR
```
diff --git a/dockers/docker-fpm-quagga/bgpcfgd b/dockers/docker-fpm-quagga/bgpcfgd
index b78f0d02..9a6cff0d 100755
--- a/dockers/docker-fpm-quagga/bgpcfgd
+++ b/dockers/docker-fpm-quagga/bgpcfgd
@@ -157,7 +157,8 @@ class BGPConfigManager(object):
                     self.lo_ipv6 = ip_addr
                     txt = self.zebra_set_src_template.render(rm_name="RM_SET_SRC6", lo_ip=ip_addr, ip_proto="v6")
                 else:
-                    syslog.syslog(syslog.LOG_ERR, "Got ambigous ip addres '%s'" % ip_addr)
+                    syslog.syslog(syslog.LOG_ERR, "Got ambiguous ip address '%s'" % ip_addr)
+                    return
             except:
                 syslog.syslog(syslog.LOG_ERR, "Error while rendering set src template '%s'" % ip_addr)
             else:

```

Now I have more changes.